src/service/api.js

```diff
@@ -0,0 +1,64 @@
+import axios from "axios";
+import config from "../app/app.config";
+
+const http = axios.create(
+    {
+        baseURL: config.uri,
+        headers: {
+            'Content-Type': 'application/json',
+        },
+    }
+)
+
+export async function logIn({username, password}) {
+    const loginResponse = await http({
+        method: "POST",
+        url: '/auth/authenticate',
+        data: {
+            username,
+            password
+        }
+    });
+    if(loginResponse.status !== 200 || !loginResponse.data.token) {
```

Comment:
Is it possible that the server returned a status code 200, but there is no token?